core/trino-main/src/main/java/io/trino/server/security/oauth2/NonceCookie.java

```diff
@@ -68,4 +67,13 @@ public static NewCookie delete()
                 true,
                 true);
     }
+
+    private static String sanitizePath(String path)
+    {
+        if (path.endsWith("/")) {
+            return path.substring(0, path.length() - 1);
+        }
```

Comment:
I don't see how this would ever happen in Trino.  If this is happening, please fix the source as this is just masking a bug.